com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAsset/UniversalRenderPipelineAssetUI.Drawers.cs

```diff
@@ -148,6 +148,21 @@ static void DrawQuality(SerializedUniversalRenderPipelineAsset serialized, Edito
             EditorGUILayout.PropertyField(serialized.hdr, Styles.hdrText);
             EditorGUILayout.PropertyField(serialized.msaa, Styles.msaaText);
             serialized.renderScale.floatValue = EditorGUILayout.Slider(Styles.renderScaleText, serialized.renderScale.floatValue, UniversalRenderPipeline.minRenderScale, UniversalRenderPipeline.maxRenderScale);
+            EditorGUILayout.PropertyField(serialized.upscalingFilter, Styles.upscalingFilterText);
+            if (serialized.asset.upscalingFilter == UpscalingFilterSelection.FSR)
```

Comment:
Hmmm, so would this mean that in order for users to see the FSR override checkbox, they'd need to select FSR upscaling, and then additionally change the render scale to something at or below 1.0? I could change it to work this way, but when dynamic resolution comes into play, do you think we'd end up changing it again since the render scale slider would become dynamic and users might not change it manually anymore?